native/packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBar.ios.js

```diff
@@ -0,0 +1,218 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { type Element, StatusBar, StyleSheet, View } from 'react-native';
+import BpkText from 'react-native-bpk-component-text';
+import { withTheme } from 'react-native-bpk-theming';
+import {
+  colorGray50,
+  colorGray100,
+  colorGray700,
+} from 'bpk-tokens/tokens/base.react.native';
+
+import {
+  type CommonTheme,
+  getThemeStyle,
+  makeThemePropType,
+  THEME_ATTRIBUTES,
+} from './common-types';
+import BpkNavigationBarBackButtonIOS from './BpkNavigationBarBackButtonIOS';
+import BpkNavigationBarTextButtonIOS from './BpkNavigationBarTextButtonIOS';
+import BpkNavigationBarIconButtonIOS from './BpkNavigationBarIconButtonIOS';
+import isIphoneX from './isIponeX';
+
+const IOS_THEME_ATTRIBUTES = [...THEME_ATTRIBUTES, 'navigationBarShadowColor'];
+
+const statusBarPadding = isIphoneX ? 44 : 20;
+
+const styles = StyleSheet.create({
+  barOuter: {
+    flexDirection: 'column',
+    paddingHorizontal: 8,
+    width: '100%',
+    backgroundColor: colorGray50,
+    shadowColor: colorGray100,
+    shadowOpacity: 1,
+    shadowRadius: 0,
+    shadowOffset: { width: 0, height: 1 },
+  },
+  barOuterWithSubtitle: {
+    paddingBottom: 16,
+  },
+  bar: {
```

Comment:
This should be renamed to `innerBar` or vice versa to match the application i nteh render method.